README.md

```diff
@@ -50,12 +50,12 @@ Seamless Communication depends on 3 libraries developed by Meta.
 fairseq2 is our next-generation open-source library of sequence modeling components that provides researchers and developers with building blocks for machine translation, language modeling, and other sequence generation tasks. All SeamlessM4T models in this repository are powered by fairseq2.
 
 ## [SONAR and BLASER 2.0](https://github.com/facebookresearch/SONAR)
-SONAR, Sentence-level multimOdal and laNguage-Agnostic Representations is a new multilingual and -modal sentence embedding space which outperforms existing sentence embeddings such as LASER3 and LabSE on the xsim and xsim++ multilingual similarity search tasks. SONAR provides text and speech encoders for many languages. SeamlessAlign was mined based on SONAR embeddings.
```

Comment:
I think this is intentional to explain where "SONAR" comes from. And we could keep multilingual as a work and "-modal" meaning multimodal.

Reply:
we could revert this line change and ship another line. Thanks!

Reply:
sure